src/lib/core/quiz.ts

```diff
@@ -384,3 +384,54 @@ export const getQuiz = async ({quizId}: {quizId: string}) => {
 
   return output
 }
+
+export const getCodeRunnerResult = async ({
+  code,
+  testRunner,
+  language,
+}: {
+  code: string
+  testRunner: string
+  language: 'javascript' | 'python' | 'c' | 'cpp' | 'java'
+}) => {
+  // edit the codeRunnerUrl according to ur origin
+  const codeRunnerUrl = 'http://localhost:3001/api/test'
+
+  const fetch = require("node-fetch")
```

Comment:
Can you use import statement instead? All import statements should be at the top.